src/components/dashboard/SnapshotWidget/index.tsx

```diff
@@ -0,0 +1,97 @@
+import { Box, Card, CardContent, Chip, Link, SvgIcon, Typography, Skeleton } from '@mui/material'
+
+import LinkIconBold from '@/public/images/sidebar/link-bold.svg'
+import useSafeSnapshot from '@/components/dashboard/SnapshotWidget/useSafeSnapshot'
+import type { SnapshotProposal } from '@/components/dashboard/SnapshotWidget/useSafeSnapshot'
+
+import css from './styles.module.css'
+
+// These colors remain the same regardless of dark/light mode
+const STATUS_TEXT_COLOR = '#FFF'
```

Comment:
do we need a constant for this? only used in the `Chip`

Reply:
I've moved it to the relevant property/CSS module.